main_test.go

```diff
@@ -138,6 +160,35 @@ OUTER:
 		`memcached_slab_mem_requested_bytes{slab="1"} 68`,
 		`memcached_slab_mem_requested_bytes{slab="5"} 194`,
 	}
+
+	if use_temp_lru == true {
+		tests = append(tests,
+			`memcached_slab_temporary_items{slab="1"}`,
+			`memcached_slab_lru_hits_total{lru="temporary",slab="5"}`,
+			`memcached_slab_temporary_items{slab="5"}`)
+	}
+
+	if strings.HasPrefix(memcache_version, "1.5.") {
```

Comment:
Good point. Originally I thought to re-evaluate the test when 1.6.x was released, but it could be a hassle long term. Also now these metrics are not going away, will try to find a better solution :)